Libraries/Components/TextInput/TextInput.js

```diff
@@ -252,6 +252,15 @@ var TextInput = React.createClass({
      */
     selectionState: PropTypes.instanceOf(DocumentSelectionState),
     /**
+     * The start and end of the text input's selection. Set start and end to
+     * the same value to position the cursor.
+     * @platform ios
+     */
+    selection: PropTypes.shape({
```

Comment:
@sahrens Seems like I haven't fully understood how to implement a controlled prop. Thanks for pointing that out. This might also be the cause for `selection`'s value fluctuating between multiple values the last time I tried this. I do still think normalizing the single line & multiline selection events would be necessary before providing a means of setting the selection, but that's more related to the PR in general, not this comment. Thanks again.